transport/http/handler.go

```diff
@@ -82,6 +87,26 @@ func (h handler) callHandler(w http.ResponseWriter, req *http.Request) error {
 
 	ctx := context.Background()
 
+	// Extract opentracing etc baggage from headers
```

Comment:
Most of the large block here from 90-108 seems like it should be factored out into it's own function, to keep this func nice and small and readable.